src/scss/components/hero-banner/_hero-banner.scss

```diff
@@ -34,235 +67,258 @@
       z-index: 10;
     }
   }
+  
```

Comment:
The banner image does a weird crop for me while signed in at smaller breakpoint. It looks like this is due to the `margin-top: -300px` applied - it seemed to look better when I removed/adjusted that margin.
Also, is this `.block--local-tasks` code block needed? It seems to work without it on my end.

Reply:
fixed.